ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/user/user.tsx

```diff
@@ -79,6 +79,11 @@ export default hot(module)(
                 showLabel={false}
                 showValidationIssues={false}
                 onChange={handlePasswordChange}
+                onKeyPress={event => {
+                  if (event.key === 'Enter') {
```

Comment:
✏️ The better way to do this is to wrap the sign in form in form tags, specify a submit button (`type="submit"` and for non-submit buttons `type="button"`), and create an on submit handler (`<form onSubmit={...}>`).

Reply:
I think @BernardIgiri 's suggestion would be the way to go here.

Reply:
I agree this is the right way to it. I did try those changes, but it was getting clobbered by other components. IMO any more effort spent on "doing it right", in this case, probably isn't worth it.

Reply:
Did you call `event.preventDefault()` in your `onSubmit` handler?

Reply:
no